gpkit/__init__.py

```diff
@@ -120,7 +120,10 @@ def __exit__(self, type_, val, traceback):
 from .geometric_program import GeometricProgram
 from .constraints.signomial_program import SignomialProgram
 from .constraints.model import Model
+from .constraints.set import ConstraintSet
+from .constraints.costed import CostedConstraintSet
```

Comment:
not sure `CostedConstraintSet` should be in `__init__.py`. Is the guidance for users to use `Model`, or `CostedConstraintSet`? The name `CostedConstraintSet` feels a bit awkward for user consumption.